Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/sdk/common/HttpObservable.java

```diff
@@ -220,6 +220,15 @@ public HttpObservable setDefaultHeaderGroup(@Nullable HeaderGroup defaultHeaders
         return this;
     }
 
+    public List<Header> getHeaderList() {
```

Comment:
>getHeaderList [](start = 24, length = 13)

Why need this? looks like the caller can do it by `Arrays.asList(getDefaultHeaders())`